src/itdelatrisu/opsu/states/SongMenu.java

```diff
@@ -372,21 +380,69 @@ public void render(GameContainer container, StateBasedGame game, Graphics g)
 
 		// song buttons
 		BeatmapSetNode node = startNode;
-		int songButtonIndex = 0;
-		if (node != null && node.prev != null) {
+		int starｔNodeOffsetoffset = 0;
+		if (node.prev != null) {
```

Comment:
You can't delete the `node != null` check here, or else you'll get a NullPointerException if the beatmap list is empty (e.g. when a search returns no results).